primitives/arithmetic/src/traits.rs

```diff
@@ -65,7 +53,54 @@ impl<T:
 	CheckedShl + CheckedShr + CheckedAdd + CheckedSub + CheckedMul + CheckedDiv +
 	Saturating + PartialOrd<Self> + Ord + Bounded +
 	HasCompact + Sized
-> SimpleArithmetic for T {}
+> BaseArithmetic for T {}
+
+/// A meta trait for arithmetic.
+///
+/// Arithmetic types do all the usual stuff you'd expect numbers to do. They are guaranteed to
+/// be able to represent at least `u32` values without loss, hence the trait implies `From<u32>`
+/// and smaller ints. All other conversions are fallible.
+pub trait RuntimeArithmetic:
```

Comment:
I think we should call it `AtLeast32Bit`. The name `RuntimeArithmetic` doesn't make sense here, as this is nothing we enforce in the runtime.

Reply:
`32BitArithmetic`?

Reply:
That is not right, because we are not doing `32bit` arithmetic here :D We just add `From<u32>` and `From<u16>`. That does say anything about the arithmetic.

Reply:
I am not fully happy with `AtLeast32Bit ` but we'll go with it for now since I cannot come up with any other name :D 🤷🏻‍♂️

Reply:
Yeah me neither :D But I also don't have any better idea :(